### PR TITLE
add binary state sensor for each window and door (open/closed)

### PR DESCRIPTION
**Pull request by prvakt**

### CODE REVIEW

File: custom_components/myskoda/translations/en.json [hunk 1]
```diff
@@ -109,6 +165,9 @@
             }
         },
         "image": {
+            "render_light_3x": {
+                "name": "Status Render of Vehicle"
+            },
```
Comment (WebSpider) [inline custom_components/myskoda/translations/en.json:170]: This does not belong in this PR, please remove it

**Review by WebSpider — CHANGES_REQUESTED** [review]
Another awesome PR! :clinking_glasses: 

One minor remark inline, and please add the sensors in `README.md` as well

File: custom_components/myskoda/translations/en.json (hunk 1 above)
Comment (prvakt) [reply custom_components/myskoda/translations/en.json:170]: sorry, it got mixed up with my test translation heh... already preparing readme update ;)

**Review by WebSpider — APPROVED** [review]
lgtm
